src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/ResourceQueries/PagingQueryString.feature

```diff
@@ -108,3 +108,30 @@ Feature: Paging Support for GET requests for Ed-Fi Resources
                         }
                     ]
                   """
+
+        @API-147
+        Scenario Outline: 12 Ensure clients can not GET information when filtering by limit and offset using invalid values
+            # Some of these are "SQL Injection" style attacks
+             When a GET request is made to "/ed-fi/schools?offset=<value>"
+             Then it should respond with 400
+              And the response body is
+                  """
+                  {
+                        "detail": "The request could not be processed. See 'errors' for details.",
+                        "type": "urn:ed-fi:api:bad-request",
+                        "title": "Bad Request",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {},
+                        "errors": [
+                            "Offset must be a numeric value greater than or equal to 0."
+                        ]
+                    }
+                  """
+        Examples:
+                  | value                    |
+                  | -1                       |
+                  | 'zero'                   |
+                  | '5; select * from users' |
+                  | '0)'                     |
+                  | '1%27'                   |
```

Comment:
This `Scenario Outline` was previously in the `TotalQueryCountString.feature` file, but it belongs here. Even then, it wasn't written correctly. This version correctly runs 5 tests with the scenario outline, each time substituting in the next value from the `Examples` table.

Reply:
Understood